src/main/kotlin/no/nav/familie/ks/sak/kjerne/behandling/steg/behandlingsresultat/BehandlingsresultatEndringUtils.kt

```diff
@@ -49,8 +49,8 @@ object BehandlingsresultatEndringUtils {
 
         val endringerForRelevantePersoner =
             relevantePersoner.any { aktør ->
-                val nåværendePersonResultatForPerson = nåværendePersonResultat.filter { it.aktør == aktør }.toSet()
-                val forrigePersonResultatForPerson = forrigePersonResultat.filter { it.aktør == aktør }.toSet()
+                val personResultat = personResultater.singleOrNull { it.aktør == aktør }
+                val forrigePersonResultat = forrigePersonResultater.singleOrNull { it.aktør == aktør }
```

Comment:
Noen spesiell grunn til at vi brukte `Set<PersonResultat>` her tidligere? Er det mulig at en aktør har to PersonResultater? @UyQuangNguyen